docs/admin/etcd.md

```diff
@@ -1,49 +1,196 @@
 ---
```

Comment:
@mml @wojtek-t can we change the name of the md file to `etcd_upgrade.md` to match the title?

I am going to revive the original generic etcd doc soon, which should be named as etcd.md.